configs/spin/resnet50_spin_pw3d.py

```diff
@@ -57,7 +57,10 @@
         in_ndc=False,
         focal_length=5000,
         image_size=(img_res, img_res),
-        principal_point=(img_res / 2, img_res / 2)))
+        principal_point=(img_res / 2, img_res / 2)),
+    quiet=True)
```

Comment:
Let's use `verbose` as the opposite of `quiet`.

Reply:
done